src/config.rs

```diff
@@ -127,6 +127,7 @@ pub enum Step {
     Restarts,
     Rtcl,
     Rustup,
+    Juliaup,
```

Comment:
nit: These are sorted alphabetically, please move it up.

Reply:
Done. Thank you.